### ISSUE TITLE
Module fails to detect branches for the branch filter

### ISSUE THREAD

**Issue by FGM** [mglaman/contribkanban.com#79]
In the case of the MongoDB module, it does display issues in the 8.x-2.x branch but does not expose that branch in the Project Branches selector.

https://contribkanban.com/board/mongodb

**Comment by mglaman** [comment]
If it's a new branch, the project needs to be refreshed. Unfortunately the only way to really do that is re-adding it, right now

**Comment by FGM** [comment]
OK, then how does one refresh the project ? I don't see anything in the UI. This branch is about 10 months old already.

**Comment by mglaman** [comment]
I'll have to look into it. I'm not sure why. I'm out of country at the moment, so give me a week or so.

**Comment by jonathan1055** [comment]
Hi,
Have we lost the Project Branch drop-down selector? I thought there used to be one, and the above comments imply we did, but right now we only get 'category' and 'priority' selectors.
Or am I mis-remembering it?
Jonathan

**Comment by mglaman** [comment]
Yes, this was a regression in the new version that I need to fix.

**Comment by hansfn** [comment]
Any chance of getting the branch filter back - or maybe rather a version filter like on drupal.org. Just "Any 7.x" and "Any 8.x" would help a lot.

**Comment by mglaman** [comment]
The problem with "8.x" and "7.x" is that Drupal.org's API is through RestWS. There is no "contains" or partial mapping support. So we can only filter directly on branches, without some post-processing after fetching issues.

The big discussion happened here: https://www.drupal.org/project/restws/issues/2392183
